package.json

```diff
@@ -21,7 +21,7 @@
     "storybook": "start-storybook -s ./public -p 9000",
     "build-storybook": "build-storybook -s ./public",
     "start": "npm run build:icons && npm run storybook",
-    "test": "jest -u --coverage"
+    "test": "jest --coverage"
```

Comment:
the `-u` flag is only meant to be used when you're sure that the snapshots should be updated, if you run it every time you'll never know when the snapshots failed

Reply:
👍